addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -238,7 +238,6 @@ def _l10n_in_edi_cancel_invoice(self, invoice):
             return {invoice: {"success": True, "attachment": attachment}}
 
     def _l10n_in_validate_partner(self, partner, is_company=False):
```

Comment:
```suggestion
    def _l10n_in_validate_partner(self, partner):
```
Not sure why using `is_company` ? Can't see any use in the method :thinking:

Reply:
Indeed, but better to do such things in master.

Reply:
Too late to see this comment